crates/rolldown_plugin/src/plugin_context.rs

```diff
@@ -67,6 +101,27 @@ impl From<PluginContextImpl> for PluginContext {
 }
 
 impl PluginContextImpl {
+  pub async fn load(
```

Comment:
It just came to me that `load` should not blocks the running task. What this PR does now is the right direction.